src/test/java/com/learning/pages/swaglabs/SwagLabsInventoryPagePageTest.java

```diff
@@ -0,0 +1,49 @@
+package com.learning.pages.swaglabs;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+import com.learning.TestsConfiguration;
+import com.learning.browsers.Browser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.Test;
+
+@ContextConfiguration(classes = TestsConfiguration.class)
+public class SwagLabsInventoryPagePageTest extends AbstractTestNGSpringContextTests {
+
+  @Autowired
+  private Browser browser;
+  @Autowired
+  private SwagLabsInventoryPage swagLabsInventoryPage;
+
+  @Test
+  void shouldObjectsNotBeNullWhenAutowired() {
+    assertNotNull(browser);
+    assertNotNull(swagLabsInventoryPage);
+  }
+
+  @Test
+  void shouldGoToShopPage() throws InterruptedException {
```

Comment:
This is not a valid system/feature test. It does something. But it's a waste of time to execute it since it barely tests anything. You basically test if you implemented PO correctly, and can use Browser.

Reply:
Just like you said. When PO and Browser frame will be ready then it will be easier to extend tests with more realistic cases.
I believe it will be next task.